### PR TITLE
fix: PropertiesPopover maxWidth changing fixed units to relative units

### PR DESCRIPTION
**Pull request by zsviczian**
resolves: #8455

### CODE REVIEW

File: packages/excalidraw/components/ColorPicker/ColorPicker.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ const ColorPickerPopupContent = ({
   return (
     <PropertiesPopover
       container={container}
-      style={{ maxWidth: "208px" }}
+      style={{ maxWidth: "17em" }}
```
Comment (dwelle) [inline packages/excalidraw/components/ColorPicker/ColorPicker.tsx:109]:
```suggestion
      style={{ maxWidth: "13em" }}
```

208/16=13

With 17em, it's this wide:

image

**Comment by zsviczian** [comment]
CSS is a pain in the ... in Obsidian 13em results in this. I wonder...
Also this is something new with the most recent Obsidian update. Oh how I love to debug stylesheet issues!

image

**Comment by zsviczian** [comment]
The solution is to use rem instead of em.

changing .App-menu__left from 200px to 12rem because of this

image

**Comment by dwelle** [comment]
Oh, didn't notice it was in `em` initially. Yeah.

Also, the left menu should be `12.5rem` instead of `12rem` to keep it the same.

**Comment by zsviczian** [comment]
sorry... I'll leave this be. I am not prepared to struggle myself through the test scripts and snapshots. I'll just modify the values to relative values in my fork.

**Comment by dwelle** [comment]
> sorry... I'll leave this be. I am not prepared to struggle myself through the test scripts and snapshots. I'll just modify the values to relative values in my fork.

Looks like just a test flake.

Merging...

**Review by dwelle — APPROVED** [review]

**Pull request merged**